src/components/MainArea.tsx

```diff
@@ -2,27 +2,25 @@ import { useEffect, useState } from "react";
 import { ShadowDom } from "./ShadowDom.tsx";
 import CreateRecipe from "./CreateRecipe.tsx";
 import Recipes from "./Recipes.tsx";
-import { Route, Routes } from "react-router-dom";
+import { Route, Routes, useLocation } from "react-router-dom";
 import CheckRecipes from "./CheckRecipes.tsx";
 
 const MainArea = () => {
   const [parentElement] = useState(() =>
-    document.querySelector('[data-gtm-section="hp-topBanner"]'),
+    document.querySelector(".sc-e03dc118-1.kvCJfT"),
```

Comment:
tohle bude nestabilni, musime vymyslet lepsi zpusob, jak neco upravi na strance, tohle se pregeneruje.

Reply:
jj, vim... chtela jsem pouzit id="pageFullWidth", to je i lepsi, pac je tam oddelena main a footer.... ale pak to zas hazi Client-side Exception Occurred